tests/ui/rust-2024/unsafe-extern-blocks/safe-unsafe-on-unadorned-extern-block.rs

```diff
@@ -0,0 +1,12 @@
+extern "C" {
+    safe static TEST1: i32;
+    //~^ ERROR items in unadorned `extern` blocks cannot have safety qualifiers
+    safe fn test1(i: i32);
+    //~^ ERROR items in unadorned `extern` blocks cannot have safety qualifiers
+}
+
+fn test2() {
+    test1(TEST1);
+}
+
+fn main() {}
```

Comment:
Should this test have the header, e.g. that would cause it to be tested in both editions, as the others do?  Also, should it set the feature flag?